server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -87,6 +87,70 @@ const virtualCardMutations = {
       return virtualCard;
     },
   },
+  createCardForCollective: {
+    description: 'Create new Stripe Virtual Card for existing hosted collective',
+    type: new GraphQLNonNull(VirtualCard),
+    args: {
+      name: {
+        type: GraphQLString,
+        description: 'Virtual Card name',
+      },
+      monthlyLimit: {
+        type: GraphQLInt,
+        description: 'Virtual Card monthly limit',
+      },
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Account where the virtual card will be associated',
+      },
+      assignee: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Individual account responsible for the card',
+      },
+    },
+    async resolve(_: void, args, req: express.Request): Promise<VirtualCardModel> {
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to assign a virtual card');
+      }
+
+      const collective = await fetchAccountWithReference(args.account, { loaders: req.loaders, throwIfMissing: true });
+      const host = await collective.getHostCollective();
+
+      if (!req.remoteUser.isAdminOfCollective(host)) {
+        throw new Unauthorized("You don't have permission to edit this collective");
+      }
+
+      const assignee = await fetchAccountWithReference(args.assignee, {
+        loaders: req.loaders,
```

Comment:
Should we throw an error?

Reply:
In which case ?

Reply:
If assignee is not found

Reply:
Could you do something here? I see that if assigned is not found, we have a crash later on `const user = await assignee.getUser();`

Reply:
`throwIfMissing: true` should be enough?

Reply:
Ok, I also added it to the assign card mutation